schedule/yast/encryption/lvm_encrypt_separate_boot_pvm.yaml

```diff
@@ -13,29 +13,15 @@ vars:
   FULL_LVM_ENCRYPT: 1
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/skip_module_registration
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_text_mode
-  - installation/partitioning/new_partitioning_gpt
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_ssh_service_enabled
-  - installation/installation_settings/open_ssh_port
-  - installation/bootloader_settings/disable_plymouth
-  - installation/disable_grub_timeout
-  - installation/start_install
-  - installation/performing_installation/perform_installation
-  - installation/logs_from_installation_system
-  - installation/performing_installation/confirm_reboot
-  - installation/handle_reboot
-  - installation/boot_encrypt
-  - installation/first_boot
-  - console/system_prepare
-  - console/force_scheduled_tasks
-  - console/validate_lvm
-  - console/validate_encrypt
+  expert_partitioning:
```

Comment:
just overwrite for now for simplicity `suggested_partitioning`

Reply:
done.